app/jobs/add_live_result_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddLiveResultJob < ApplicationJob
+  self.queue_adapter = :shoryuken if Rails.env.production? && !EnvConfig.WCA_LIVE_SITE?
+  queue_as EnvConfig.LIVE_QUEUE if Rails.env.production? && !EnvConfig.WCA_LIVE_SITE?
+
+  def perform(params)
+    ActionCable.server.broadcast("results_#{params[:round_id]}",
```

Comment:
Also, please share this broadcast key (perhaps in the same module as the "WCA Live enabled?" feature above)